File: Java+/Spring+/spring-java13-example/src/test/java/app/mvc/HelloWorldControllerTest.java
package app.mvc;

import app.BaseMvcTest;
import org.junit.jupiter.api.Test;
import org.springframework.http.MediaType;

import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;

class HelloWorldControllerTest extends BaseMvcTest {

    @Test
    void hello() throws Exception {
        mvc.perform(
                get(HelloWorldController.ENDPOINT)
                        .contentType(MediaType.APPLICATION_JSON)
        )
                .andExpect(status().isOk())
                .andExpect(content().string(HelloWorldController.CONTENT));
    }
}

File: Kotlin+/KotlinSpringExample/src/test/kotlin/app/mvc/HelloWorldControllerTest.kt
package app.mvc

import app.BaseMvcTest
import org.junit.jupiter.api.Test
import org.springframework.http.MediaType
import org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get
import org.springframework.test.web.servlet.result.MockMvcResultMatchers.content
import org.springframework.test.web.servlet.result.MockMvcResultMatchers.status

class HelloWorldControllerTest : BaseMvcTest() {

    @Test
    fun requestParam() {
        mvc.perform(get(HelloWorldController.ENDPOINT).contentType(MediaType.APPLICATION_JSON))
                .andExpect(status().isOk)
                .andExpect(content().string(HelloWorldController.CONTENT))
    }
}
